src/docs/sdk/envelopes.mdx

```diff
@@ -465,6 +465,26 @@ The item contains a user feedback / user report JSON payload:
 
 *None*
 
+### Client Report
+
+Item type `"client_report"` contains a client report payload encoded in JSON.
+
+See the <Link to="/sdk/client-reports/">client reports</Link> documentation for the payload
+details.
+
+**Constraints:**
+
+- This Item may occur at most once per Envelope.
+- This Item can either be included in an Envelope with other Items, or it may be sent by itself.
+
+**Envelope Headers:**
+
+*None*
```

Comment:
I don't believe a client report sent by itself would require any specific envelope headers, since there's no event id specifically associated with the report, and the timestamp is already in the report payload.  Correct?

Reply:
afaik this is correct